doc/source/howtos/crash-recovery.rst

```diff
@@ -0,0 +1,295 @@
+.. _pxc-crash-recovery:
+
+================================================================================
+Crash Recovery
+================================================================================
+
+Unlike the standard MySQL replication, a |pxc| cluster acts like one logical entity,
+which controls the status and consistency of each node as well as the status
+of the whole cluster. This allows maintaining the data integrity more efficiently than
+with traditional asynchronous replication without losing safe writes
+on multiple nodes at the same time.
+
+However, there scenarios where the database service can stop with no node being
+able to serve requests.
+
+.. contents::
+   :local:
+
+.. _pxc-crash-recovery-scenario-1:
+
+Scenario: Node A is gracefully stopped
+================================================================================
+
+In a three node cluster (node A, Node B, node C), one node (node A, for example)
+is gracefully stopped: for the purpose of maintenance, configuration change,
+etc.
+
+In this case, the other nodes receive a "good bye" message from the stopped node
+and the cluster size is reduced; some properties like quorum calculation or auto
+increment are automatically changed. As soon as node A is started again, it
+joins the cluster based on its :variable:`wsrep_cluster_address` variable in
+:file:`my.cnf`.
+
+If the the writeset cache (:variable:`gcache.size`) on nodes B and/or C still
+has all the transactions executed while node A was down, joining is possible via
+:term:`IST`. If :term:`IST` is impossible due to missing transactions in donor’s
+gcache, the fallback decision is made by the donor and :term:`SST` is started
+automatically.
+
+.. _pxc-crash-recovery-scenario-2:
+
+Scenario: Two nodes are gracefully stopped
+================================================================================
+
+Similar to :ref:`pxc-crash-recovery-scenario-1`, the cluster size is reduced to
+1 --- even the single remaining node C forms the primary component and is able to
+serve client requests. To get the nodes back into the cluster, you just need
+to start them.
+
+However, node C will be switched to “Donor/Desynced” state as it has to provide
```

Comment:
This is a little confusing.  You should mention that another node is joining the cluster and then node C goes into donor/desynced.  Maybe somethng like "However, when a new node joins the cluster, node C will be switched to..."